src/routes/files.go

```diff
@@ -27,11 +27,13 @@ func FilesRoutes(router *gin.Engine, database *db.Database) {
 	})
 
 	router.POST("/files/upload", func(c *gin.Context) {
-		file, err := c.FormFile("file")
+		form, err := c.MultipartForm()
 		if err != nil {
-			c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"status": "no file uploaded"})
+			c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"status": "no file(s) uploaded"})
```

Comment:
use proper capitalization in the status message. "status" itself is a variable, and the current camelCase is good